Adding onboarding docs and configuring live preview (#5333)
ROCm/rocm-libraries: projects/hipdnn/docs/user-guides/install/hipdnn-prerequisites.rst

.. meta::

>   :description: Learn about the prerequisites and dependencies required to install hipDNN.

Reviewer: The description isn't accurate. 
The file should also be renamed to better align with the contents (install topic).

  :keywords: hipDNN, ROCm, install, prerequisites

.. _prerequisites:

*******************
hipDNN installation
*******************

> The hipDNN frontend API is distributed as a header-only library; it requires a *development* ROCm package installation.

Reviewer 2: ```suggestion
The hipDNN frontend API is distributed as a header-only library; it requires a development ROCm package installation.
```

> The hipDNN frontend API is distributed as a header-only library; it requires a *development* ROCm package installation.

Reviewer: ```suggestion
The hipDNN frontend API is distributed as a header-only library. It requires a development ROCm package installation.
```

These development packages contain the ``dev`` prefix.

.. important::

>   The base ROCm install package does *not* include the hipDNN frontend API header files. You must use the development ROCm packages.

Reviewer: ```suggestion
  The base ROCm install package doesn't include the hipDNN frontend API header files. You must use the development ROCm packages.
```

>   For example, on Linux with a gfx950 GPU, install the ``amdrocm-core-dev7.12-gfx950`` package instead of the ``amdrocm7.12-gfx950`` package (``amdrocm-core-dev7.12-gfx950`` includes the ``amdrocm7.121-gfx950`` package, so only the ``amdrocm-core-dev7.12-gfx950`` package needs to be installed).

Reviewer 2: Are amdrocm7.12-gfx950 and amdrocm7.121-gfx950 separate packages or is one of those a typo?

Author: @CMiservaAMD can you confirm?

>   For example, on Linux with a gfx950 GPU, install the ``amdrocm-core-dev7.12-gfx950`` package instead of the ``amdrocm7.12-gfx950`` package (``amdrocm-core-dev7.12-gfx950`` includes the ``amdrocm7.12-gfx950`` package, so only the ``amdrocm-core-dev7.12-gfx950`` package needs to be installed).

Reviewer: ```suggestion
  For example, on Linux with an AMD Instinct MI350X Series GPU, install the ``amdrocm-core-dev7.12-gfx950`` package instead of the ``amdrocm7.12-gfx950`` package (``amdrocm-core-dev7.12-gfx950`` includes the ``amdrocm7.12-gfx950`` package, so only the ``amdrocm-core-dev7.12-gfx950`` package needs to be installed).
```
